src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -10,16 +10,18 @@ import Button from '../../../../components/button/Button';
 import PrimaryButton from '../../../../components/primary-button/PrimaryButton';
 import MenuToggle from '../../../../components/dropdown-menu/MenuToggle';
 import { Flyout, Overlay } from '../../../../components/flyout';
+import { AND, OR } from '../../constants';
 
 import messages from '../../messages';
 
-import type { ColumnType } from '../../flowTypes';
+import type { ColumnType, SelectOptionType } from '../../flowTypes';
 
 type State = {
     appliedConditions: Array<Object>,
     areErrorsEnabled: boolean,
     conditions: Array<Object>,
     isMenuOpen: boolean,
+    selectedConnector: typeof AND | typeof OR,
```

Comment:
Shouldn't the type be "ConnectorType"? It isn't clear to me why use typeof here.

Reply:
`typeof AND` will just resolve to string here. so you can either set it to values (`selectedConnector: AND | OR`) or just use string.

Reply:
I'd like to see new flow types AND / OR and a new ```ConnectorType = AND | OR```.

One challenge though is with naming. Will both flowTypes and constants.js export AND? My experience with flow is limited here.

Reply:
So long as we have the literal value as the type, it will do the right thing. looks like `AND` and `OR` have "value-types" which makes this evaluate properly.

<img width="298" alt="screen shot 2019-02-19 at 12 00 37 pm" src="https://user-images.githubusercontent.com/3056447/53043761-09e30380-343e-11e9-8296-2c533f3569ad.png">

Reply:
Thanks for pointing it out. 

Then maybe all we need is a new ConnectorType = typeof AND | typeof OR. Thoughts?

Reply:
That looks good to me